test/bats/tests/config/config_tsa.json

```diff
@@ -0,0 +1,101 @@
+{
+    "store": {
+        "version": "1.0.0",
+        "plugins": [
+            {
+                "name": "oras",
+                "cosignEnabled": true,
+                "useHttp": true
+            }
+        ]
+    },
+    "policy": {
+        "version": "1.0.0",
+        "plugin": {
+            "name": "configPolicy"
+        }
+    },
+    "verifier": {
+        "version": "1.0.0",
+        "plugins": [
+            {
+                "name": "schemavalidator",
+                "artifactTypes": "application/vnd.aquasecurity.trivy.report.sarif.v1",
+                "schemas": {
+                    "application/sarif+json": "file:///home/runner/.ratify/schemas/sarif-2.1.0-rtm.5.json"
+                }
+            },
+            {
+                "name": "sbom",
+                "artifactTypes": "application/spdx+json",
+                "nestedReferences": "application/vnd.cncf.notary.signature",
+                "disallowedLicenses":["AGPL"],
+                "disallowedPackages":[{"name":"log4j-core","versionInfo":"2.13.0"}]
+            
+            },
+            {
+                "name": "cosign",
+                "artifactTypes": "application/vnd.dev.cosign.artifact.sig.v1+json",
+                "key": ".staging/cosign/cosign.pub"
+            },
+            {
+                "name": "notation",
+                "artifactTypes": "application/vnd.cncf.notary.signature",
+                "verificationCerts": [
+                    "~/.config/notation/localkeys/ratify-bats-test.crt",
+                    ".staging/tsa/tsaroot.cer"
+                ],
+                "trustPolicyDoc": {
+                    "version": "1.0",
+                    "trustPolicies": [
+                        {
+                            "name": "default",
+                            "registryScopes": [
+                                "*"
+                            ],
+                            "signatureVerification": {
+                                "level": "strict",
+                                "verifyTimestamp": "afterCertExpiry"
+                            },
+                            "trustStores": [
+                                "ca:certs",
+                                "tsa:certs"
```

Comment:
I see that we are defining the same `certs` store with both tsa and ca type. Does this config mean that notation will trust the `ca` verification certs as `tsa` certs too for verification? (and vice-versa)

Reply:
I feel we should separate certs for ca and tsa which is recommended by notation spec though it's supported for now.